src/api/goods.js

```diff
@@ -3,9 +3,18 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll() {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error(`${response.status} - ${response.statusText}`);
+      }
+
+      return response.json();
+    });
 }
 
-export const get5First = () => {};
+export const get5First = result => result.sort(
+  (a, b) => (a.name).localeCompare(b.name),
+).slice(0, 5);
```

Comment:
Let's make your app more consistent.
`get5First` function should not be used with some abstract `result`.
It is better to explicitly call `getAll()` here and return result of sorting and slicing.